src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -405,6 +408,18 @@ public void toggleLocationServices() {
         execute(TOGGLE_LOCATION_SERVICES);
     }
 
+    public void toggleFlightMode() {
```

Comment:
I think it would be cool if new methods were defined at the HasNetworkConnection interface